src/__stories__/logo-story.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {BlauLogo, Logo, MovistarLogo, O2Logo, TelefonicaLogo, VivoLogo} from '../logo';
+import {StorySection} from './helpers';
+import {Text2} from '../text';
+
+export default {
+    title: 'Components/logo',
+    component: Logo,
+    parameters: {
+        fullScreen: true,
+    },
+};
+
+type Args = {logoType: 'isotipo' | 'imagotype' | 'vertical'; size: number};
```

Comment:
can you add inverse state control? In [text link story](https://mistica-csnam7wn6-tuentisre.vercel.app/?path=/story/components-text-link--default) you have an example